x-pack/heartbeat/monitors/browser/suite.go

```diff
@@ -0,0 +1,108 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package browser
```

Comment:
This file isn't new, it's just moved. Enough of it changed due to renaming the `ss` var to `s` that git thinks it's all new.